### PR TITLE
Add rails 5 to travis and make gem compatible with rails 5

### PR DESCRIPTION
**Pull request by iffyuva**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,29 @@
 language: ruby
 
 rvm:
-  - 1.9.3
   - 2.0.0
-  - 2.1.6
-  - 2.2.2
+  - 2.1.10
```
Comment (emilsoman) [inline .travis.yml]: Maybe we can support only Ruby 2.2+ from this version on ?

File: .travis.yml [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
+  - 2.2.5
+  - 2.3.1
 
 gemfile:
-  - Gemfile
   - gemfiles/Gemfile.rails-3.2.x
   - gemfiles/Gemfile.rails-4.0.x
   - gemfiles/Gemfile.rails-4.1.x
   - gemfiles/Gemfile.rails-4.2.x
```
Comment (emilsoman) [inline .travis.yml]: Shall we remove everything before 4.1 ?

File: app/controllers/rapidfire/questions_controller.rb [hunk 3]
```diff
@@ -67,5 +67,13 @@ def find_question!
     def index_location
       rapidfire.question_group_questions_url(@question_group)
     end
+
+    def question_params
+      if Rails::VERSION::MAJOR == 4 || Rails::VERSION::MAJOR == 5
```
Comment (emilsoman) [inline app/controllers/rapidfire/questions_controller.rb]: Maybe we should consider removing support for Rails 3.x

File: gemfiles/Gemfile.rails-3.2.x [hunk 4]
```diff
@@ -4,3 +4,4 @@ source "http://rubygems.org"
 
 gemspec path: '..'
 gem 'rails', "~> 3.2.13"
+gem 'test-unit', '~> 3.0'
```
Comment (emilsoman) [inline gemfiles/Gemfile.rails-3.2.x]: Why is test-unit required ?

File: gemfiles/Gemfile.rails-5.0.x [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# -*- ruby -*-
+
+source 'http://rubygems.org'
+
+gemspec path: '..'
+gem 'rails', '~> 5.0.0'
```
Comment (emilsoman) [inline gemfiles/Gemfile.rails-5.0.x]: Should we add a Gemfile for Rails latest master ?

File: gemfiles/Gemfile.rails-3.2.x (hunk 4 above)
Comment (iffyuva) [inline gemfiles/Gemfile.rails-3.2.x]: on latest versions of ruby, test unit is removed. so i explicity added that gem for rails 3.x

File: gemfiles/Gemfile.rails-5.0.x (hunk 5 above)
Comment (iffyuva) [inline gemfiles/Gemfile.rails-5.0.x]: nope, not required. we generally check with latest version of rails, thats it.

**Comment by iffyuva** [comment]
@emilsoman lets not pull out support for rails 3, or ruby 2.0, or 2.1 if we hit a blocker, we will consider removing support

**Pull request merged**